modules/rds/main.tf

```diff
@@ -1,84 +1,172 @@
+resource "aws_db_subnet_group" "postgres_subnet_group" {
+  ## REQUIRED
+  subnet_ids = var.lambda_subnet_ids
+
+  ## OPTIONAL
+  description = "ORCA PostgreSQL Subnet group where the ORCA database resides."
+  name        = "${var.prefix}-postgres-subnet-group"
+  tags        = local.tags
+}
+
+
+## postgresql - PostgreSQL database created for ORCA metadata/status
+## =============================================================================
 # https://blog.faraday.io/how-to-create-an-rds-instance-with-terraform/
+# https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance
+# https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/latest
 resource "aws_db_instance" "postgresql" {
+  ## REQUIRED
+  allocated_storage = 20
+  engine            = "postgres"
+  identifier        = "${var.prefix}-postgres-sndbx"
+  instance_class    = "db.t3.micro"
+  password          = var.postgres_user_pw
+  username          = "postgres"
+
+  ## OPTIONAL
   # set apply_immediately true just for testing
-  apply_immediately = false
-  allocated_storage          = 20
-  # max_allocated_storage      = 21
-  engine                     = "postgres"
-  engine_version             = "11"
-  identifier                 = "${var.prefix}-postgres-sndbx"
-  # Can't do micro and storage_encrypted
-  instance_class             = "db.t2.micro"
-  name                       = "postgres"
-  password                   = var.postgres_user_pw
-  username                   = "postgres"
-  iam_database_authentication_enabled = "true"
-  backup_retention_period    = 7
-  backup_window              = "03:00-06:00"
-  maintenance_window         = "Mon:00:00-Mon:03:00"
-  # auto_minor_version_upgrade = "${var.auto_minor_version_upgrade}"
-  port                       = var.database_port
-  vpc_security_group_ids     = [var.vpc_postgres_ingress_all_egress_id]
-  db_subnet_group_name       = aws_db_subnet_group.postgres_subnet_group.id
-  # parameter_group_name       = "default.postgres11"
-  storage_encrypted          = false
-  deletion_protection        = false
-  skip_final_snapshot        = true
-  final_snapshot_identifier  = "daacx"
-  enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
+  apply_immediately                   = false
+  auto_minor_version_upgrade          = true
+  backup_retention_period             = 7
+  backup_window                       = "03:00-06:00"
+  db_subnet_group_name                = aws_db_subnet_group.postgres_subnet_group.id
+  delete_automated_backups            = true
+  deletion_protection                 = false
+  enabled_cloudwatch_logs_exports     = ["postgresql", "upgrade"]
+  engine_version                      = "11"
+  final_snapshot_identifier           = "daacx"
+  iam_database_authentication_enabled = true
+  maintenance_window                  = "Mon:00:00-Mon:03:00"
+  name                                = "postgres"
```

Comment:
maybe we can also move some of these parameters like backup_retention_period, engine_version, final_snapshot_identifier to variables.tf 
Thoughts?

Reply:
yes we can then keep them in the main.tf for now then since our customers don't want that and since the RDS instance will go away.